services/db.js

```diff
@@ -52,8 +52,8 @@ export class MongoDB {
     async create (collectionName, data){
         try{
             const collection = this.db.collection(collectionName);
-            const doc = await collection.insertOne(data);
-            return doc;
+            await collection.insertOne(data);
```

Comment:
An asynchronous function must return its result which is a `Promise` that can be resolved to some value.